### ISSUE TITLE
Description of VIM-GitGutter

### ISSUE THREAD

**Issue by ijmitch** [ZOSOpenTools/meta#740]
I made the typo change for its name in #739, but then reading the [project's description](https://github.com/airblade/vim-gitgutter) I think the paragraph saying:

> vim-gitgutter is a plugin that allows VIM users on z/OS to interact with Git repositories directly from within the editor.

is overstating its function.

As the name implies, `vim-gitgutter` is actually just improving working with changed code locally with annotations in the 'gutter' of the editor session. I cannot see that it claims to work with actual commits or pushes/pulls/fetches and the like.

Am I missing something?

**Comment by ijmitch** [comment]
"to interact with Git repositories directly from within the editor" probably means something more like [vim-fugitive](https://github.com/tpope/vim-fugitive).

**Comment by IgorTodorovskiIBM** [comment]
You're correct, it seems to only provide support for annotations. Feel free to edit or enhance with vim-fugitive. I have yet to try that one
